documents/knowledge/elements.yaml

```diff
@@ -3752,6 +3752,62 @@
     str, num: zfill(a, b)
   vectorise: false
 
+- element: "∆ċ"
+  name: Nth Cardinal
+  arity: 1
+  description: Get the nth cardinal / convert number to words
+  overloads:
+    num: num_to_words(a)
+  vectorise: true
+  tests:
+    - "[0] : 'zero'"
+    - "[1] : 'one'"
+    - "[2] : 'two'"
+    - "[3] : 'three'"
+    - "[4] : 'four'"
+    - "[5] : 'five'"
+    - "[6] : 'six'"
+    - "[7] : 'seven'"
+
+- element: "∆o"
+  name: Nth Ordinal
+  arity: 1
+  description: Get the nth ordinal / convert number to wordth ordinal
+  overloads:
+    num: num_to_ordinal(a)
+  vectorise: true
+  tests:
+    - "[0] : 'zeroth'"
+    - "[1] : 'first'"
+    - "[2] : 'second'"
+    - "[3] : 'third'"
+    - "[4] : 'fourth'"
+    - "[5] : 'fifth'"
+    - "[6] : 'sixth'"
+    - "[7] : 'seventh'"
+
+- element: "∆M"
+  name: Mode
+  arity: 1
+  description: Get the mode of a list
+  overloads:
+    lst: mode(a)
+  vectorise: false
+  tests:
+    - "[[1, 1, 1, 1, 2, 2, 3, 3, 3, 4]] : [1]"
+    - "[[1, 1, 1, 1, 2, 2, 3, 3, 3, 4, 4, 4, 4]] : [1]"
+
+- element: "∆ṁ"
+  name: Median
+  arity: 1
+  description: Get the median of a list
+  overloads:
+    lst: median(a)
+  vectorise: false
+  tests:
+    - "[[1, 2, 3, 4, 5]] : 3"
+    - "[[1, 2, 3, 4, 5, 6]] : [3, 4]"
```

Comment:
Hrm, perhaps we could note in the description that this doesn't return the average of the middle two values if there's an even number of values but rather returns both of them in a list?

Reply:
Or - get this - it could just correctly implement median by returning the average of those two?

Reply:
@pxeger but then what about strings? how do you average two strings?

Reply:
@pxeger I feel like this is more useful behavior in case you want to get the lower/higher element. It's trivial to average them after that.

Reply:
correct - `ṁ` exists to average a list

Reply:
But if you don't know whether it's gonna be two or one, then `ṁ` won't help (because it returns 0 when given a list).

Reply:
clarification made.